packages/console/src/assets/docs/guides/web-next-app-router/index.ts

```diff
@@ -9,7 +9,7 @@ const metadata: Readonly<GuideMetadata> = Object.freeze({
   target: ApplicationType.Traditional,
```

Comment:
the description looks weird? should be something like App Router is a new feature in Next.js 13 bluh bluh

<img width="1036" alt="image" src="https://github.com/logto-io/logto/assets/14722250/af951038-7024-44e6-83d2-fce7f1fa888c">

Reply:
Yeah... CoPilot is not as ellegant as we expected. We should review these descriptions tomorrow

Reply:
updated